pkg/config/setup/process_test.go

```diff
@@ -304,6 +308,12 @@ func TestEnvVarOverride(t *testing.T) {
 			value:    "datacat.com",
 			expected: "datacat.com",
 		},
+		{
+			key:      "process_config.ignore_zombie_processes.enabled",
+			env:      "DD_PROCESS_CONFIG_IGNORE_ZOMBIE_PROCESSES",
+			value:    "false",
+			expected: false,
```

Comment:
[bug] The env var key you're using here doesn't match the one you've set. Also this test is to verify setting a non-default value works. In this case that would mean setting and expecting it to be `true`.